<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SNpreScanDialog</class>
 <widget class="QDialog" name="SNpreScanDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pre Scan Dialog</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Tooling1</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tooling</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Base</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Module base</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Module</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Tooling_1">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;機箱序號:&lt;/p&gt;&lt;p&gt;FX-[產品型號]-[1~3位數字,流水號]&lt;/p&gt;&lt;p&gt;PS: 此序號會跟Base_Setting.ini裡比對,找不到會錯誤&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Base_1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;機箱定位座序號&lt;/p&gt;&lt;p&gt;1~3位數字&lt;/p&gt;&lt;p&gt;PS:需要跟Module base 相同&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ModuleBase_1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;模組座序號:&lt;/p&gt;&lt;p&gt;1~3位數字&lt;/p&gt;&lt;p&gt;PS:需要跟base 相同&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Module_1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;模組序號:&lt;/p&gt;&lt;p&gt;FX-[產品型號+M]-[1~3位數字,流水號]&lt;/p&gt;&lt;p&gt;PS: 此序號會用產品型號跟機箱產品型號作比對,不相同會錯誤&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Tooling2</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tooling</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Base</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Module base</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Module</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Tooling_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;機箱序號:&lt;/p&gt;&lt;p&gt;FX-[產品型號]-[1~3位數字,流水號]&lt;/p&gt;&lt;p&gt;PS: 此序號會跟Base_Setting.ini裡比對,找不到會錯誤&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Base_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;機箱定位座序號&lt;/p&gt;&lt;p&gt;1~3位數字&lt;/p&gt;&lt;p&gt;PS:需要跟Module base 相同&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ModuleBase_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;模組座序號:&lt;/p&gt;&lt;p&gt;1~3位數字&lt;/p&gt;&lt;p&gt;PS:需要跟base 相同&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Module_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;模組序號:&lt;/p&gt;&lt;p&gt;FX-[產品型號+M]-[1~3位數字,流水號]&lt;/p&gt;&lt;p&gt;PS: 此序號會用產品型號跟機箱產品型號作比對,不相同會錯誤&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Tooling3</string>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tooling</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Base</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Module base</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Module</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Tooling_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;機箱序號:&lt;/p&gt;&lt;p&gt;FX-[產品型號]-[1~3位數字,流水號]&lt;/p&gt;&lt;p&gt;PS: 此序號會跟Base_Setting.ini裡比對,找不到會錯誤&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Base_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;機箱定位座序號&lt;/p&gt;&lt;p&gt;1~3位數字&lt;/p&gt;&lt;p&gt;PS:需要跟Module base 相同&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ModuleBase_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;模組座序號:&lt;/p&gt;&lt;p&gt;1~3位數字&lt;/p&gt;&lt;p&gt;PS:需要跟base 相同&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Module_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;模組序號:&lt;/p&gt;&lt;p&gt;FX-[產品型號+M]-[1~3位數字,流水號]&lt;/p&gt;&lt;p&gt;PS: 此序號會用產品型號跟機箱產品型號作比對,不相同會錯誤&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_Tooling_1</tabstop>
  <tabstop>lineEdit_Base_1</tabstop>
  <tabstop>lineEdit_ModuleBase_1</tabstop>
  <tabstop>lineEdit_Module_1</tabstop>
  <tabstop>lineEdit_Tooling_2</tabstop>
  <tabstop>lineEdit_Base_2</tabstop>
  <tabstop>lineEdit_ModuleBase_2</tabstop>
  <tabstop>lineEdit_Module_2</tabstop>
  <tabstop>lineEdit_Tooling_3</tabstop>
  <tabstop>lineEdit_Base_3</tabstop>
  <tabstop>lineEdit_ModuleBase_3</tabstop>
  <tabstop>lineEdit_Module_3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
